api/rs/slint/mcu.md

```diff
@@ -55,12 +45,18 @@ In the snippet above, three features are selected:
    This macro is only available in the Rust Standard Library (std), but not in bare metal environments. As a fallback, the `unsafe-single-threaded`
    feature changes Slint to use unsafe static for storage. This way, you guarantee to use Slint API only from a single thread, and not from interrupt handlers.
  * `libm`: We select this feature to enable the use of the [libm](https://crates.io/crates/libm) crate to provide traits and functions for floating point arithmetic.
-   They are typically provided by the Rust Standard Library (std), but that is not available in bare metal environments.
+   They're typically provided by the Rust Standard Library (std), but that's not available in bare metal environments.
 
 It might be necessary to enable the [Feature resolver version 2](https://doc.rust-lang.org/cargo/reference/features.html#feature-resolver-version-2)
-in your Cargo.toml if you notice that your dependencies are attempting to build with `std` support even if it was disabled.
+in your Cargo.toml if you notice that your dependencies are attempting to build with `std` support even when disabled.
 This is the default when using the Rust 2021 Edition or later.
 
+Then add the `slint-build` crate as a build dependency:
+
+```sh
+cargo add slint-build@1.0.0 --build
```

Comment:
I'm curious, is there a specific reason to pass `--build` as the last argument instead of after `add`?